R/rd-author.R

```diff
@@ -0,0 +1,7 @@
+
+#' @export
```

Comment:
Given how simple this is, I think you could could keep it in `rd-markdown.R`

Reply:
I agree this is quite simple, but `rd-markdown.R` has such homogenous content I felt bad to blemish it with this one. 
Besides, all `merge.xxx()` functions belong in a specific `rd-xxx.R` file.
Are you sure you want me to put it in `rd-markdown.R`?

Reply:
Yes 😄